wp_cli/src/wp_cli_posts.rs

```diff
@@ -4,6 +4,8 @@ use wp_serde_helper::deserialize_i64_or_string;
 
 use crate::run_wp_cli_command;
 
+const POST_FIELDS_ARG: &str = "--fields=ID, post_title, post_date, post_status, post_author, post_date_gmt, post_content, post_excerpt, comment_status, ping_status, post_password, post_modified, post_modified_gmt, guid, post_type";
```

Comment:
I missed the spaces in this argument. Having them doesn't impact this PR and I am removing them in the next PR, so it can be safely ignored.